### ISSUE TITLE
add show method for compatibility reason with other engines

### ISSUE THREAD

**Issue by djouallah** [GlareDB/glaredb#1397]
it will be really nice if the method show is available, so i can easily switch Python code between different engines
show print the results of a sql query

```
import glaredb 
con = glaredb.connect()
con.sql(f""" select * FROM parquet_scan('./1/lineitem/00.parquet') """).show()
```

**Comment by scsmithr** [comment]
This will be going out in our next release.

Added in https://github.com/GlareDB/glaredb/pull/1315

**Comment by greyscaled** [comment]
Going to mark as closed, but as @scsmithr stated it won't be available until our next release, which given the feature set will be `v0.3.0`

**Comment by greyscaled** [comment]
Hey @djouallah :wave: just dropping by to say we've released `0.3.0` with `show`

https://github.com/GlareDB/glaredb/releases/tag/v0.3.0

**Comment by djouallah** [comment]
no, it does not work in colab
